pkg/cmd/taskrun/describe_test.go

```diff
@@ -731,3 +731,99 @@ func TestTaskRunWithSpecDescribe_custom_timeout(t *testing.T) {
 	}
 	golden.Assert(t, actual, fmt.Sprintf("%s.golden", t.Name()))
 }
+
+func TestTaskRunDescribe_last(t *testing.T) {
+	clock := clockwork.NewFakeClock()
+
+	/*
+		list.GetAllTaskRuns( ) returns a slice of strings containing all taskruns starting from the most recent one,
+		hence here the recent one is kept as the first element of the slice `trs`
+	*/
+	trs := []*v1alpha1.TaskRun{
+		tb.TaskRun("tr-1", "ns",
+			tb.TaskRunStatus(
+				tb.TaskRunStartTime(clockwork.NewFakeClock().Now().Add(20*time.Second)),
+				tb.StatusCondition(apis.Condition{
+					Type:   apis.ConditionSucceeded,
+					Status: corev1.ConditionTrue,
+				}),
+				tb.StepState(
+					cb.StepName("step1"),
+					tb.SetStepStateTerminated(reasonCompleted),
+				),
+				tb.StepState(
+					cb.StepName("step2"),
+					tb.SetStepStateTerminated(reasonCompleted),
+				),
+			),
+			tb.TaskRunSpec(
+				tb.TaskRunTaskRef("t1"),
+				tb.TaskRunInputs(tb.TaskRunInputsParam("input", "param")),
+				tb.TaskRunInputs(tb.TaskRunInputsParam("input2", "param2")),
+				tb.TaskRunInputs(tb.TaskRunInputsResource("git", tb.TaskResourceBindingRef("git"))),
+				tb.TaskRunInputs(tb.TaskRunInputsResource("image-input", tb.TaskResourceBindingRef("image"))),
+				tb.TaskRunOutputs(tb.TaskRunOutputsResource("image-output", tb.TaskResourceBindingRef("image"))),
+				tb.TaskRunOutputs(tb.TaskRunOutputsResource("image-output2", tb.TaskResourceBindingRef("image"))),
+			),
+		),
+		tb.TaskRun("tr-2", "ns",
+			tb.TaskRunStatus(
+				tb.TaskRunStartTime(clockwork.NewFakeClock().Now().Add(2*time.Minute)),
+				tb.StatusCondition(apis.Condition{
+					Type:   apis.ConditionSucceeded,
+					Status: corev1.ConditionTrue,
+				}),
+				tb.StepState(
+					cb.StepName("step3"),
+					tb.SetStepStateTerminated(reasonCompleted),
+				),
+				tb.StepState(
+					cb.StepName("step4"),
+					tb.SetStepStateTerminated(reasonCompleted),
+				),
+			),
+			tb.TaskRunSpec(
+				tb.TaskRunTaskRef("t1"),
+				tb.TaskRunInputs(tb.TaskRunInputsParam("input", "param")),
+				tb.TaskRunInputs(tb.TaskRunInputsParam("input2", "param2")),
+				tb.TaskRunInputs(tb.TaskRunInputsResource("git", tb.TaskResourceBindingRef("git"))),
+				tb.TaskRunInputs(tb.TaskRunInputsResource("image-input", tb.TaskResourceBindingRef("image"))),
+				tb.TaskRunOutputs(tb.TaskRunOutputsResource("image-output", tb.TaskResourceBindingRef("image"))),
+				tb.TaskRunOutputs(tb.TaskRunOutputsResource("image-output2", tb.TaskResourceBindingRef("image"))),
+			),
+		),
+	}
+
+	cs, _ := test.SeedTestData(t, pipelinetest.Data{
+		TaskRuns: trs,
+		Namespaces: []*corev1.Namespace{
+			{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "ns",
+				},
+			},
+		},
+	})
+
+	version := "v1alpha1"
```

Comment:
We should use `v1beta1` here as it is the most recent version of the Tekton API.